components/builder-jobsrv/src/server/log.rs

```diff
@@ -0,0 +1,48 @@
+use error::Result;
+use std::fs;
+use std::path::PathBuf;
+
+/// Encapsulates the local filesystem directory in which in-process
+/// build job logs will be collected prior to being sent to long-term
+/// storage.
+#[derive(Clone, Debug)]
+pub struct LogDirectory {
+    path: PathBuf
+}
+
+impl LogDirectory {
+
```

Comment:
V. minor nit - the de-factor formatting pattern in the codebase is to not have a blank line after the impl line. Feel free to ignore :)